packages/hub/prisma/migrations/20240928161949_workflow_format/migration.sql

```diff
@@ -0,0 +1,5 @@
+-- AlterTable
+ALTER TABLE "AiWorkflow" ADD COLUMN     "format" INTEGER NOT NULL DEFAULT 2;
+
+-- old workflows
+UPDATE "AiWorkflow" SET "format" = 1;
```

Comment:
Is the idea that "format" is a version number? Can this maybe instead be a string that's a clearer version number? 

It seems pretty annoying to support all the old versions, forever. I imagine we could delete them after a few weeks or so. 

Maybe we could adjust the workflow row to save the prompt and final result - then those at least would be consistent, even if the format changes. So we don't promise to save the whole workflow past a week or so, just the input and output.